docs/loot/recipes.md

```diff
@@ -0,0 +1,625 @@
+
+<CodeHeader>#/minecraft:recipe_shapeless/result</CodeHeader>
+```json
+{
+	"item": "minecraft:spawn_egg",
+	"data": "query.get_actor_info_id('minecraft:chicken')"
+}
+```
+
+The optional integer `"count"` property may be used to stack items. It defaults to `1`. Currently, setting the count only functions in [crafting](#crafting) and [heating](#heating) recipe outputs and [shapeless recipe ingredients](#ingredients). A provided count is ignored in other locations.
+
+::: tip NOTE
+If a count greater than `1` is provided for an item that does not stack, an error will be thrown. There is no way to force single-return recipe outputs, like those in shapeless recipes or brewing mixes, to return multiple items in one transaction.
+:::
+
+#### Identifier Additions
+Additional identifiers not typically usable are available to recipes to describe basic potions.
+
+<CodeHeader>#/minecraft:recipe_brewing_mix/input</CodeHeader>
+```json
+"minecraft:potion_type:strength"
+```
+
+These identifiers are not usable in the object notation, only the string notation. Variants are unavailable for splash and lingering potions. All such identifiers follow the format: <code>minecraft:potion_type:<em>potion_effect</em></code>, where <code><em>potion_effect</em></code> can be one of the following:
+
+- `water`
+- `awkward`
+- `mundane`
+- `thick`
+- `healing`
+- `regeneration`
+- `swiftness`
+- `strength`
+- `harming`
+- `poison`
+- `slowness`
+- `weakness`
+- `water_breathing`
+- `fire_resistance`
+- `nightvision`
+- `invisibility`
+- `leaping`
+- `slow_falling`
+- `turtle_master`
+
+Where supported, `long_` and `strong_` prefixes may be used to designate modified potions, such as `minecraft:potion_type:strong_poison`.
+
+## Crafting
+Crafting operations instantly transform inputs to outputs using crafting grids. Two crafting recipe types are available: [shapeless recipes](#shapeless-recipes), whose inputs may be arranged in any way, and [shaped recipes](#shaped-recipes), used to define strict arrangements of inputs.
+Crafting recipes support both crafting tables and stonecutters:
+
+<CodeHeader>#/minecraft:recipe_shapeless/</CodeHeader>
+```json
+"tags": ["crafting_table", "stonecutter"]
+```
+
+`"crafting_table"` applies to both vanilla crafting tables and the player 2 × 2 crafting grid in their inventory. There is currently no way to opt into one but not the other. Crafting recipes additionally support custom tags, linking recipes to a [crafting grid provided by a custom block](https://wiki.bedrock.dev/blocks/blocks-16.html#minecraft-crafting-table).
```

Comment:
Wiki links need to be made relative. Remove this part: `https://wiki.bedrock.dev`